src/game.cpp

```diff
@@ -6607,11 +6419,16 @@ static void setMapScroll()
 
 // -----------------------------------------------------------------------------------------
 /*returns the current type of save game being loaded*/
-UDWORD getSaveGameType()
+GAME_TYPE getSaveGameType()
 {
 	return gameType;
 }
 
+UDWORD getSaveGameVersion()
```

Comment:
Bit of a hack to make moving the two methods less painful. Exposes the save game version so that `levels.cpp` has access to it.